src/app/components/login-page/login-page.component.css

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
CSS isn't necessary for now, but we can use some of it for stuff just for basics